workers/redirector.js

```diff
@@ -32,14 +32,14 @@ const experimentPages = [
         'sampleRate': 0.54
     },
     {
-        'targetPath': `/en-DE/`,
-        'sandboxPath': `/en-DE/exp/`,
-        'sampleRate': 0.30
+        'targetPath': `/de/`,
```

Comment:
@duallain I realized that this may not work because `workerPaths` has en-US in the paths?